pkgs/applications/science/machine-learning/vowpal-wabbit/default.nix

```
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchgit, cmake, boost169, zlib }:
+
+let
+  version = "8.9.2";
+in
+stdenv.mkDerivation {
```

```suggestion
stdenv.mkDerivation rec {
```

This will allow self-reference, you won't have to declare `version` in an `let ... in` block anymore.

---

Sounds good, will make that change